contracts/oracles/sources/oracle.move

```diff
@@ -0,0 +1,38 @@
+module oracles::oracle {
+
+    use sui::object::{Self, UID};
+    use sui::tx_context::{Self,TxContext};
+    use sui::transfer;
+
+    use SupraOracle::SupraSValueFeed::{Self, OracleHolder};
+
+    struct Storage has key, store {
+        id: sui::object::UID,
+        supra_price: u256,
+    }
+
+    fun init(ctx:&mut TxContext) {
+        transfer::share_object(
+            Storage {
+                id:object::new(ctx),
+                supra_price:0,
+            }
+        );
+    }
+
+    public fun set_supra_price(oracle_holder:&OracleHolder, storage: &mut Storage) {
+        let (sui_usd_price,_,_,_) = SupraSValueFeed::get_price(oracle_holder, 90);
```

Comment:
Move `90` to a constant. It can be defined at the start of module.

Something like: `SUPRA_SUI_USD_PAIR: u32 = 90`